Lib/ldap/ldapobject.py

```diff
@@ -532,7 +522,7 @@ def delete_ext(self,dn,serverctrls=None,clientctrls=None):
         form returns the message id of the initiated request, and the
         result can be obtained from a subsequent call to result().
     """
-    dn = self._bytesify_input(dn)
+    dn = self._bytesify_input('dn', dn)
```

Comment:
``if PY2:``?

Reply:
`_bytesify_input` already contains a check. I only added the `PY2` check when there were several calls to `_bytesify_input`.